tests/python/tank_test/mock_appstore.py

```diff
@@ -31,7 +31,11 @@
 
 from tank import TankError
 from tank.platform.environment import InstalledEnvironment
-from distutils.version import LooseVersion
+from tank.util import suppress_known_deprecation
+try:
+    from setuptools._distutils.version import LooseVersion
+except ImportError:
```

Comment:
I have a preference for "IF six.py3XX" (using the oldest supported version) than ImportError in these cases.

Not a blocker but I think it'll be easier to maintain too.

Reply:
I'm not able to use SIX for that since it only has a constant for `PY38`. In this case, I need to check 3.10 so I used `sys.version_info` instead.